File: Java Script - Introduction to Programming/5.1. Loops/08. Odd Even Sum.js
function oddEvenSum(arg) {
  arg = arg.map(Number)
  let [odd, even] = [0, 0]

  for (var i = 1; i <= arg[0]; i++) {
    i % 2 == 0 ? even += arg[i] : odd += arg[i]
  }

  if (even === odd)
    return `Yes\nSum = ${even}`
  return `No\nDiff = ${Math.abs(even - odd)}`
}

//console.log(oddEvenSum([4, 10, 50, 60, 20]))

File: Pyrhon - Introduction to Programming/5.1. Loops/08. Odd Even Sum.py
lst = [int(input())  for _ in range(int(input()))]
odd_sum = sum(lst[::2])
even_sum = sum(lst[1::2])

print(f'Yes\nSum = {even_sum}' if even_sum == odd_sum else f'No\nDiff = {abs(even_sum - odd_sum)}')


'''
4
10
50
60
20
#yes
'''
